operator/v1/tests/clustercsidrivers.operator.openshift.io/AWSEFSDriverVolumeMetrics.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "ClusterCSIDriver"
+crdName: clustercsidrivers.operator.openshift.io
+featureGate: AWSEFSDriverVolumeMetrics
+tests:
+  onCreate:
+  - name: Should be able to create a minimal ClusterCSIDriver
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: ClusterCSIDriver
+      metadata:
+        name: efs.csi.aws.com
+      spec: {} # No spec is required for a ClusterCSIDriver
+    expected: |
+      apiVersion: operator.openshift.io/v1
+      kind: ClusterCSIDriver
+      metadata:
+        name: efs.csi.aws.com
+      spec:
+        logLevel: Normal
+        operatorLogLevel: Normal
+  - name: Should be able to create ClusterCSIDriver with AWS EFS volume metrics disabled
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: ClusterCSIDriver
+      metadata:
+        name: efs.csi.aws.com
+      spec:
+        driverConfig:
+          driverType: AWS
+          aws:
+            efsVolumeMetrics:
+              state: Disabled
+        logLevel: Normal
+        operatorLogLevel: Normal
+    expected: |
+      apiVersion: operator.openshift.io/v1
+      kind: ClusterCSIDriver
+      metadata:
+        name: efs.csi.aws.com
+      spec:
+        driverConfig:
+          driverType: AWS
+          aws:
+            efsVolumeMetrics:
+              state: Disabled
+        logLevel: Normal
+        operatorLogLevel: Normal
+  - name: Should be able to create ClusterCSIDriver with AWS EFS volume metrics enabled and default options
```

Comment:
None of the tests presently actually test the defaulting, I think if you had one that had an empty `options: {}` you would see that the defaulting you have today doesn't work as you want it to

Reply:
Removed the defaulting from the schema. We'll handle it in the code.